src/dp_vnf.c

```diff
@@ -125,6 +125,8 @@ int dp_add_vnf(const uint8_t ul_addr6[DP_VNF_IPV6_ADDR_SIZE], enum dp_vnf_type t
 	vnf->type = type;
 	vnf->vni = vni;
 	vnf->port_id = port_id;
+
+	memset(&vnf->alias_pfx, 0, sizeof(vnf->alias_pfx));
```

Comment:
You can just do what you are doing everywhere else, i.e. direct initialization using `struct dp_vnf = {}` which should initialize to zero